### PR TITLE
 Added documentation for Visual tabs

### LINKED ISSUE #183: Document visual tabs
see `Vt` command.

### PR DESCRIPTION
**Pull request by officialcjunior**
**Detailed description**
* Added command hints for `doc` and `doof`

* Added documentation about Visual tabs. Fixes #183 

My radare2 version:
```
radare2 4.5.0-git 24888 @ linux-x86-64 git.4.4.0-328-gcff5ec573
commit: cff5ec5736fc6fa33fc8bc3ea4b2c7e26e9603ac build: 2020-06-24__21:56:49
```

### CODE REVIEW

**Comment by officialcjunior** [comment]
Removed the WIP tag.

There's also some .snippets lying around. Can I add them to .gitignore too?

**Comment by wargio** [comment]
Yes please.

**Comment by officialcjunior** [comment]
Added Vim Snippets to .gitignore.

Pull Request is now ready for review.

File: src/analysis/variables.md [hunk 1]
```diff
@@ -35,7 +37,7 @@ If we check the help for `afvr` we will get the way two others commands works to
 | afvrj                       return list of register arguments in JSON format
 | afvr- [name]                delete register arguments at the given index
 | afvrg [reg] [addr]          define argument get reference
-| afvrs [reg] [addr]          define argument set reference
+| afvrs [reg] [addr]          define argument set referencev
```
Comment (wargio) [inline src/analysis/variables.md:40]: hmm remove that `v`

Comment (officialcjunior) [reply src/analysis/variables.md]: Apologies. 
Fixed that.

**Comment by alex-bender** [comment]
> Removed the WIP tag.
> 
> There's also some .snippets lying around. Can I add them to .gitignore too?

@officialcjunior 
There is a local .gitignore file. `The .git/info/exclude file has the same format as any .gitignore file`

**Pull request merged**